handbook/quality/README.md

```diff
@@ -0,0 +1,146 @@
+
+### Inbox
+When a new bug is created using the [bug report form](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=bug%2C%3Areproduce&template=bug-report.md&title=), it is in the "inbox" state. 
+At this state, QA is responsible for going through the inbox and asking for more reproduction details from the reporter, asking the product team for more guidance, or acknowledging the bugs.
+
+> Some bugs may also be the domain of the digital-experience team. If QA believes this is the case, then QA should put the bug onto the g-digital-experience board and assign it to ... TODO. The digital experience team has their own bug process which is not governed by this process.
+
+### Weekly bug review
+QA has weekly check-in with product to go over the inbox items. QA is responsible for proposing “not a bug”, closing due to lack of response (with a nice message), or raising other relevant questions. All requires product agreement
+
+Requesters have six weeks to provide follow-up information for each info request. We will ping them again as a reminder at three weeks. After six weeks, we will close the bug to remove it from our visibility, but requesters are welcome to re-open and provide context.
+
+QA may also propose that a reported bug is not actually a bug. A bug is defined as “behavior which is not according to spec or implied by spec.” Thereafter, it is assigned to the relevant product manager for them to decide on its priority.
+
+### Acknowledging bugs
+Otherwise, QA should apply the acknowledged state to the bug. QA has one week to reproduce the bug.
+
+Once reproduced, QA should document the reproduction steps and move it to the reproduced state.
+
+### Reproduced
+When reproduced, the assigned engineering manager is responsible for investigating root cause of the bug and proposing solutions to their product counterpart if it requires discussion. Otherwise, the EM includes it in this release if there is space or the next release.
+
+### After reproduced
+After it is in a release formally, the bug should be treated like any other piece of work per the standard engineering process.
+
+### Fast track for Fleeties
+Fleeties do not have to wait for QA to reproduce the bug. If you are confident it is reproducible, is a bug, and the reproduction steps are well-documented, it can be moved directly to the reproduced state.
+
+### During release testing
+When release is in testing, QA should use the the slack channel #help-release to keep everyone aware of issues found. All bugs related to a release should be reported in the channel after creating the bug first.
+
+In the release channel, product may decide whether the bug is a release blocker. Release blockers must be fixed before a release can be cut.
+
+### Critical bugs
+A critical bug is defined as: “a bug that causes users to be unable to use a workflow, upgrade Fleet, or causes irreversible damage such as loss of data.”
```

Comment:
Heads up, there are several definitions of "critical" bugs in the docs:
- Patch release worthy bugs: https://fleetdm.com/docs/contributing/releasing-fleet#patch-releases
- Release blocking bugs: https://fleetdm.com/docs/contributing/releasing-fleet#release-freeze-period

I think we can define "what is a critical bug?" in one place and link to this place from each separate section. This way, we're not maintaining slightly different variations of the same "critical bug" concept.

What do you think?

Reply:
Good find thank you.